ownership-chain/pallets/parachain-staking/src/lib.rs

```diff
@@ -697,7 +697,18 @@ pub mod pallet {
 	impl<T: Config> BuildGenesisConfig for GenesisConfig<T> {
 		fn build(&self) {
 			assert!(self.blocks_per_round > 0, "Blocks per round must be > 0");
-			<InflationConfig<T>>::put(self.inflation_config.clone());
+
+			// Set inflation configuration
+			let mut inflation_config = self.inflation_config.clone();
+			// if all the round values are 0, derive them from the annual values
+			if inflation_config.round.min.is_zero() &&
```

Comment:
IMO, this check is not needed at all. By always setting round values from annual we also prevent invalid input to for `round` values, i.e with this change if we can put `round` values to any non-zero value and it will set them. For example, annual - `10%` and round `20%` is a valid value here

Reply:
we won't the user to set the round values. The correct approach is to remove them from the configuration. It's not a good solution but it's the only one I can figure out to tell the user : "Do not use the round"

Reply:
> we won't the user to set the round values
do you mean `we _want_ the user to set the round values` or `we _won't allow_ the user to set the round values`?
if former, then it would be possible set any arbitrary value for `round`, if latter, then with this current logic you are allowing to set any `round` value as long as it is a non-zero. 

I am suggesting to remove the check for zero and always set the round values from `annual`, regardless if `round` values were provided or not (basically discard it). this will be compatible with the future logic of not providing `round` as well

Reply:
ok we disagree

Reply:
well, I am trying to understand but it's hard:

> we won't the user to set the round values

does it mean we want to allow or not?

and with the current logic, is it possible to set `round` values as long as any of them is `non-zero`? How is it preventing or taking care of case when some non-zero value is passed inadvertently?